libfreerdp/emu/scard/smartcard_emulate.c

```diff
@@ -1253,7 +1253,7 @@ HANDLE WINAPI Emulate_SCardAccessStartedEvent(SmartcardEmulationContext* smartca
 	WLog_Print(smartcard->log, smartcard->log_default_level, "SCardAccessStartedEvent {");
 
 	/* Not required, return random */
-	winpr_RAND(&hEvent, sizeof(hEvent));
+	winpr_RAND((void*)&hEvent, sizeof(hEvent));
 
```

Comment:
Strange I would have bet that cast to `void *` was implicit in C.

Reply:
No, the only place these casts added are `void**` to `void*` or similar casts.

While implicit like integer narrow/widen it is (as the title suggests) often a source for buggy code (pass wrong type to function) and for that reason not necessarily without warnings